host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -1558,6 +1691,159 @@ static void destroy_camera_component(RASPIVID_STATE *state)
 }
 
 /**
+ * Create the splitter component, set up its ports
+ *
+ * @param state Pointer to state control struct
+ *
+ * @return MMAL_SUCCESS if all OK, something else otherwise
+ *
+ */
+static MMAL_STATUS_T create_splitter_component(RASPIVID_STATE *state)
+{
+   MMAL_COMPONENT_T *splitter = 0;
+   MMAL_PORT_T *splitter_output = NULL;
+   MMAL_ES_FORMAT_T *format;
+   MMAL_STATUS_T status;
```

Comment:
MMAL_STATUS_T status = MMAL_ENOSYS;
Otherwise we can return an uninitialised value from the if (state->camera_component == NULL) error path.